rastervision_pytorch_backend/rastervision/pytorch_backend/examples/object_detection/cowc_potsdam.py

```diff
@@ -36,47 +76,84 @@ def make_scene(id):
                 raster_uri,
                 crop_uri,
                 label_uri=label_uri,
-                size=1000,
+                size=2000,
                 min_features=5)
             raster_uri = crop_uri
 
         raster_source = RasterioSourceConfig(
             uris=[raster_uri], channel_order=[0, 1, 2])
 
+        vector_source = GeoJSONVectorSourceConfig(
+            uri=label_uri, default_class_id=0, ignore_crs_field=True)
         label_source = ObjectDetectionLabelSourceConfig(
-            vector_source=GeoJSONVectorSourceConfig(
-                uri=label_uri, default_class_id=0, ignore_crs_field=True))
+            vector_source=vector_source)
 
         return SceneConfig(
             id=id, raster_source=raster_source, label_source=label_source)
 
     class_config = ClassConfig(names=['vehicle'])
-    chip_sz = 300
-    dataset = DatasetConfig(
+    scene_dataset = DatasetConfig(
         class_config=class_config,
         train_scenes=[make_scene(id) for id in train_ids],
         validation_scenes=[make_scene(id) for id in val_ids])
-    chip_options = ObjectDetectionChipOptions(neg_ratio=5.0, ioa_thresh=0.9)
-    predict_options = ObjectDetectionPredictOptions(
-        merge_thresh=0.5, score_thresh=0.9)
+
+    if nochip:
+        window_opts = {}
+        # set window configs for training scenes
+        for s in scene_dataset.train_scenes:
+            window_opts[s.id] = ObjectDetectionGeoDataWindowConfig(
+                # method=GeoDataWindowMethod.sliding,
+                method=GeoDataWindowMethod.random,
+                size=img_sz,
+                # stride=img_sz // 2,
+                size_lims=(200, 300),
+                # h_lims=(200, 300),
+                # w_lims=(200, 300),
+                max_windows=500,
+                max_sample_attempts=100,
+                ioa_thresh=0.75,
```

Comment:
I think we should make the nochip `ioa_thresh` and `neg_ratio` match the values that we were using when chipping. The performance is pretty sensitive to these options.